### ISSUE TITLE
Undefined reference to __pthread_mutex_init

### ISSUE THREAD

**Issue by jubalh** [rr-debugger/rr#3199]
On openSUSE Tumblweed with gcc 11.2 I get:

```
/usr/bin/ld.bfd: /home/abuild/rpmbuild/BUILD/rr-5.5.0/src/preload/overrides.c:55: undefined reference to `__pthread_mutex_init'
```

I tried to look through existing bugs but couldn't find a solution.

Full build log available [here](https://build.opensuse.org/build/devel:tools/openSUSE_Tumbleweed/x86_64/rr/_log)

**Comment by Keno** [comment]
Presumably this is a continuation of https://github.com/rr-debugger/rr/issues/2913. Maybe try extending the fix from that issue?

**Comment by jubalh** [comment]
Honestly I'm not sure where to start or what to do.

**Comment by rocallahan** [comment]
Fixed in d8b1db03360df84c4e369b2858a21dc6feee3213

**Comment by jubalh** [comment]
Confirmed. Thanks @rocallahan
